src/Pdf/index.ts

```diff
@@ -72,6 +77,13 @@ export async function getPdfData(query: PdfQuery): Promise<PdfData> {
 			destinationIdsWithPlaces.get(destinationId)!
 		);
 
```

Comment:
This inner map function is becoming quite long and I have to read it thru to find out what it does. I suggest to refactor it to some function and call the function instead so other colleague could find out what it does from then name of the function. Something like `fetchDestinationsData`